server/handlers/getters.go

```diff
@@ -151,25 +147,53 @@ func (ah *ActivityHandler) getOrCreateSyntheticUser(userID, displayName string)
 
 			break
 		}
+
+		preferences := model.Preferences{model.Preference{
+			UserId:   u.Id,
+			Category: model.PreferenceCategoryNotifications,
+			Name:     model.PreferenceNameEmailInterval,
+			Value:    "0",
+		}}
+		if prefErr := ah.plugin.GetAPI().UpdatePreferencesForUser(u.Id, preferences); prefErr != nil {
+			ah.plugin.GetAPI().LogError("Unable to disable email notifications for new user", "UserID", u.Id, "error", prefErr.Error())
+		}
 	}
 
-	if err = ah.plugin.GetStore().SetUserInfo(u.Id, userID, nil); err != nil {
+	if err = ah.plugin.GetStore().SetUserInfo(u.Id, user.ID, nil); err != nil {
 		ah.plugin.GetAPI().LogError("Unable to link the new created mirror user", "error", err.Error())
 	}
+
 	return u.Id, err
 }
 
 func (ah *ActivityHandler) getChatChannelID(chat *msteams.Chat) (string, error) {
 	userIDs := []string{}
 	for _, member := range chat.Members {
-		mmUserID, err := ah.getOrCreateSyntheticUser(member.UserID, member.DisplayName)
+		msteamsUser, clientErr := ah.plugin.GetClientForApp().GetUser(member.UserID)
+		if clientErr != nil {
+			ah.plugin.GetAPI().LogError("Unable to get the MS Teams user", "error", clientErr.Error())
+			continue
+		}
+
+		if msteamsUser.Type == msteamsUserTypeGuest && !ah.plugin.GetSyncGuestUsers() {
+			if mmUserID, _ := ah.getOrCreateSyntheticUser(msteamsUser, false); mmUserID != "" && ah.isRemoteUser(mmUserID) {
```

Comment:
what is the second variable here if it is a error please handle this or atlease log

Reply:
There's no need to handle the error here.